subsys/bluetooth/host/att.c

```diff
@@ -2261,6 +2271,7 @@ int bt_att_send(struct bt_conn *conn, struct net_buf *buf, bt_conn_tx_cb_t cb,
 
```

Comment:
Github doesn't let me select the first if-branch for commenting, but this is about the `if (!conn ||!buf)`. With this change that does not make sense anymore as a runtime check, since it will inevitably lead to a buffer leak. Either that check should be converted in to an assert (which seems like the right choice for me) or then net_buf_unref() should be added to that branch as well.

Reply:
Yep, lets make it an assert then.

Reply:
Fixed.